build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+mainClassName = 'sh.calaba.screenshot.ScreenshotTaker'
+
+sourceSets {
+    intTest {
+        java.srcDir file('src/intTest/java')
+        compileClasspath += sourceSets.main.output + configurations.testRuntime
+        runtimeClasspath += output + compileClasspath
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'lib', include: '*.jar')
+    testCompile 'junit:junit:4.12'
+}
+
+repositories {
+    jcenter()
+}
+
+jar {
+    from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+
+    manifest {
+        attributes 'Main-Class': 'sh.calaba.screenshot.ScreenshotTaker'
+    }
+}
+
+task integrationTest(type: JavaExec) {
+    def imageFromAppFileName = 'screenshotTakerTest.png'
+    def imageFromAdbFileName = 'adbScreenshotTest.png'
```

Comment:
It is better to add description for task to get something in `gradle tasks` output.

Now it looks like:
```
Other tasks
-----------
integrationTest

Rules                
```